### ISSUE TITLE
Document the command line tools

### ISSUE THREAD

**Issue by rgov** [stp/stp#342]
It looks like the docs just talk about the C and Python APIs, and don't talk about the command line tools.

**Comment by mnswdhw** [comment]
Yes, I was also looking for the command line arguments in the documentation but could not find any. Please specify the command line tools in the documentation.
